packages/core/index.d.ts

```diff
@@ -35,6 +35,7 @@ declare module '@nivo/core' {
 
     export type Theme = Partial<{
         background: string
+        fontSize: string | 0
```

Comment:
Actually there are other props missing from this definition:

```typescript
    fontFamily: string
    textColor: string
```

It would be great if you could add them too!